core-js/src/main/javascript/cdf/components/ToggleButtonBaseComponent.js

```diff
@@ -90,7 +90,7 @@ define(['../lib/jquery', './InputBaseComponent'], function($, InputBaseComponent
         input.appendTo(li);
         li.append($("<label/>")
           .attr({'for': myself.name + i})
-          .text(myArray[i][1]));
+          .html(myArray[i][1]));
```

Comment:
I don't see anywhere in the .html jquery documentation (http://api.jquery.com/html/) where it encodes special characters, is this actually happening?